frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -104,11 +104,25 @@ const footerVariantStyles = {
   `,
 };
 
-function getParameterPanelBackgroundColor(theme?: DisplayTheme) {
+function getParameterPanelBackgroundColor(
+  theme: DisplayTheme | undefined,
+  isSticky: boolean,
+) {
   if (theme === "night") {
-    return color("bg-black");
+    return `color-mix(in srgb, var(--mb-color-bg-black), var(--mb-color-embed-bg-color-override, var(--mb-color-bg-black))  ${
```

Comment:
It's [specced here](https://metaboat.slack.com/archives/C063Q3F1HPF/p1718362852193529?thread_ts=1718215529.624989&cid=C063Q3F1HPF). This is how we achieve sticky filters on transparent background, previously we disable sticky filters entirely on transparent background.

`--mb-color-embed-bg-color-override` would be `transparent` if `background: false`

The same idea applies to all cases below.